lib/turn.rb

```diff
@@ -0,0 +1,71 @@
+class Turn
+    attr_reader :player1, :player2, :spoils_of_war, :winner
+    def initialize(player1, player2)
+      @player1 = player1
+      @player2 = player2
+      @spoils_of_war = []
+      @winner = winner
+    end
+
+    def type
+        if player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0) && player1.deck.rank_of_card_at(2) == player2.deck.rank_of_card_at(2)
+            :mutually_assured_destruction
+        elsif player1.deck.rank_of_card_at(0) != player2.deck.rank_of_card_at(0)
+            :basic
+        elsif player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0)
+          :war 
+        end
+    end
+
+  def winner
+        if type == :basic
+            if player1.deck.rank_of_card_at(0) >= player2.deck.rank_of_card_at(0)
```

Comment:
```suggestion
            if player1.deck.rank_of_card_at(0) > player2.deck.rank_of_card_at(0)
```
I think we want this just to be a `>`, not `>=` because if it were equal it would be a war situation. Same for line 29.